src/card.ts

```diff
@@ -0,0 +1,112 @@
+'use strict'
+
+import type {
+  Marqeta,
+  MarqetaOptions,
+} from './'
+
+export interface ExpirationOffset {
+  unit?: string;
+  value?: bigint;
+}
+
+export interface AddressType {
+  firstName?: string;
+  middleName?: string;
+  lastName?: string;
+  address1: string;
+  address2?: string;
+  city?: string;
+  state?: string;
+  zip?: string;
+  country?: string;
+  phone?: string;
+  postalCode?: string;
+}
+
+export interface Shipping {
+  method?: string;
+  returnAddress?: AddressType;
+  recipientAddress?: AddressType;
+  carOfLine?: string;
+}
+
+export interface NameLineType {
+  [value: string]: string;
+}
+
+export interface Text {
+  nameLine1: NameLineType;
+  nameLine2?: NameLineType
+  nameLine3?: NameLineType
+}
+
+export interface Card {
+  name?: string
+  thermalColor?: string;
+}
+
+export interface ImagesCarrier {
+  name?: string
+  message1?: string;
+}
+
+export interface NameType {
+  name?: string
```

Comment:
missing semi-colon on the end of the line... and I'm wondering if this is a useful type to have - one element?